calc/vxlan_resolver.go

```diff
@@ -127,10 +128,12 @@ func (c *VXLANResolver) OnBlockUpdate(update api.Update) (_ bool) {
 	} else {
 		// Block has been deleted. Clean up routes that were contributed by this block.
 		routes := c.blockToRoutes[key]
-		routes.Iter(func(item interface{}) error {
-			c.withdrawRoute(item.(vxlanRoute))
-			return nil
-		})
+		if routes != nil {
```

Comment:
Route can be missing if you get a duplicate delete event, which is possible if the first delete was actually a validation error.  Not sure if it's possible end-to-end with the watcher cache layer, now but it was easier to fix than to prevent the calc graph FV suite from generating this case automatically.